runtime/catalog-karapace/src/test/java/io/aklivity/zilla/runtime/catalog/karapace/internal/EventIT.java

```diff
@@ -56,4 +62,19 @@ public void shouldLogEvents() throws Exception
     {
         k3po.finish();
     }
+
+    @Test
+    public void shouldLogRemoteRegistryEvents()
+    {
+        EngineContext context = mock(EngineContext.class);
+        when(context.clock()).thenReturn(Clock.systemUTC());
+        when(context.supplyEventWriter()).thenReturn(mock(MessageConsumer.class));
+        KarapaceEventContext events = new KarapaceEventContext(context);
+        events.onUnretrievableSchemaSubjectVersion(0L, "subject", "version");
+        events.onRetrievableSchemaSubjectVersion(0L, "subject", "version");
+        events.onUnretrievableSchemaSubjectVersionStaleSchema(0L, "subject", "version", 1);
+        events.onUnretrievableSchemaId(0L, 1);
+        events.onRetrievableSchemaId(0L, 1);
+    }
+
```

Comment:
This should be done as an IT with scripts verifying each different possible behavior of the remote Karapace system.

Is anything missing from `test` binding that would let the `karapace` catalog be tested that way?

Note: please see `filesystem` catalog EventIT.